spec/fixtures/proofing/lexis_nexis/ddp/request.json

```diff
@@ -17,7 +17,8 @@
   "policy": "test-policy",
   "service_type": "all",
   "session_id": "UNIQUE_SESSION_ID",
-  "ssn_hash": "15e2b0d3c33891ebb0f1ef609ec419420c20e320ce94c65fbc8c3312448eb225",
+  "national_id_number": "123456789",
```

Comment:
(not a blocker) do you think this would be easier to keep in sync if we removed it as a fixture file and inlined it in the spec?

Reply:
I guess a little, because it's moderately easier to determine what specs to run if the data is right in the relevant spec